### ISSUE TITLE
New messaging / route broadcasting makes it nearly impossible to run demo with many ledgers + connectors

### ISSUE THREAD

**Issue by emschwartz** [interledgerjs/five-bells-demo#40]
If you try running the demo with `DEMO_NUM_LEDGERS=35` and `DEMO_NUM_CONNECTORS=5`, it takes a very long time to get it set up, seemingly because of the route broadcasting. Not sure if this is fixable right now but it used to work to be able to show a larger graph of ledgers and connectors locally.

**Comment by sentientwaffle** [comment]
You're right, the slowness is most likely due to the routes propagating. I'll try and think of a fix.

**Comment by QingjingJing** [comment]
Has this been fixed? Or is route propagation that difficult of a thing to implement?

**Comment by momerath42** [comment]
The situation has been improved; this project should be updated to use more recent versions of ilp components.
